docs/shared/reference/nx-json.md

```diff
@@ -149,6 +156,25 @@ Another target default you can configure is `outputs`:
 }
 ```
 
+When defining any options or configurations inside of a target default, you may use the `{workspaceRoot}` and `{projectRoot}` tokens. This is useful for defining things like the outputPath or tsconfig for many build targets:
+
+```json {% fileName="nx.json" %}
```

Comment:
there is no description of matching being done by target & executor and how the priority works

Reply:
This part is mentioned at the beginning of the `Target Defaults` section, but I'll add an example here as well.